packages/theme/src/theme/_navs.scss

```diff
@@ -73,41 +73,42 @@
 	}
 
 	&-tabs {
+		background-color: tint($brand-primary, 90);
```

Comment:
Is there a way to create some global values for these tint in variables.scss?
Like https://github.com/Talend/ui/blob/master/packages/theme/src/theme/_variables.scss#L21